disk_image_test.go

```diff
@@ -69,3 +70,66 @@ func TestFindDiskImage(t *testing.T) {
 	}
 
 }
+
+func TestListDiskImagesV2(t *testing.T) {
```

Comment:
No need for a V2 (and V2 doesn't tell us what it does), so this should replace the existing test.

Also, it doesn't actually check if the k3s one is excluded because `k3s` isn't on line 76. There should be a record returned with `k3s` from the API and this test should ensure civogo excludes it.

Reply:
Ok. I've updated it.